clients/scheduler/src/components/time-slot-select/index.tsx

```diff
@@ -53,48 +78,48 @@ export const TimeSlotSelect = () => {
     const minDate = new Date(slots[0].start)
     const maxDate = new Date(slots[slots.length - 1].start)
 
-    return {minDate, maxDate}
+    return { minDate, maxDate }
   }, [providerTimeSlots])
 
   useEffect(() => {
-    if (typeof maxDate === "undefined" || date >= maxDate) {
+    if (typeof maxDate === 'undefined' || date >= maxDate) {
```

Comment:
Do we want to change the quote format across the project? Is this the existing eslint config for the project? 

If we want this to get back into upstream ever, it doesn't seem like we want to go all out on linting to our own code style. If we are cool with just keeping our own fork forever then go all out!

Reply:
I think this code is due to a developer on the canvas side who wasn't setup properly. Nick.... Oh Nick...

FWIW, I don't care about linting at this point in time. When it is more appropriate, we can properly lint this repository. 

Also, the configuration in the package.json suggests single quotes for prettier.

```
"prettier": {
    "semi": false,
    "singleQuote": true,
    "arrowParens": "avoid"
  },
```